subprojects/model-core/src/main/java/org/gradle/internal/reflect/DefaultTypeValidationContext.java

```diff
@@ -50,7 +53,7 @@ private DefaultTypeValidationContext(@Nullable Class<?> rootType, boolean report
     }
 
     public static boolean onlyAffectsCacheableWork(String type) {
-        return MISSING_NORMALIZATION_ANNOTATION.equals(type);
+        return new ProblemCategory(VALIDATION + SEPARATOR + MISSING_NORMALIZATION_ANNOTATION).toString().equals(type);
```

Comment:
💅 Should we do this calculation only once? I.e. extract the missing normalization category to a static final field?